src/layout/mod.rs

```diff
@@ -2387,7 +2388,9 @@ impl<W: LayoutElement> Layout<W> {
             }
         }
     }
+}
 
+impl Layout<Mapped> {
```

Comment:
Dragging `ipc_workspaces` out of the `impl<W: LayoutElement>` block isn't super pretty, but without this, I can't access any window information here as far as I can tell